test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -626,6 +655,126 @@ public void notSettingStatusDoesntClearStatus() {
         verify(account, never()).setStatus(any());
     }
 
+    @Test
+    public void updateParticipantEmailCannotBeChanged() {
```

Comment:
Do we have a test case for when they update the participant but don't change the email and/or phone?

Reply:
Added (on update, should not touch the setters or send an email).